### PR TITLE
Updated Changelog

### PR DESCRIPTION
**Pull request by KSchlobohm**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,14 @@ All notable changes to the .NET Upgrade Assistant will be documented in this fil
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Current
+### Added
+- Bundled latest version of try-convert which adds support for WPF and Unit Test projects written with Visual Basic #403
```
Comment (sunandabalu) [inline CHANGELOG.md:8]: Could you reword this to just add support for WPF and unit test projects written with VB?

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,14 @@ All notable changes to the .NET Upgrade Assistant will be documented in this fil
...
+
+### Fixed
+- Fixed issue surfacing from floating package references. #371
```
Comment (sunandabalu) [inline CHANGELOG.md:11]: Also could you add actual links to issues/prs here? just # <number> unfortunately does not work in files in a repo :(

Comment (KSchlobohm) [reply CHANGELOG.md]: added actual links

**Review by sunandabalu — APPROVED** [review]
